common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -184,6 +184,23 @@ cACTION_APMANAGER_STEERING_EVENT_AUTH_FAIL_NOTIFICATION:
   _type: class
   params: sSteeringEvAuthFail
 
+cACTION_APMANAGER_WIFI_CREDENTIALS_SET_REQUEST:
+  _type: class
+  ssid:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SSID_MAX_LENGTH" ]
+  pass:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_PASS_MAX_LENGTH" ]
+  auth_type:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SECURITY_TYPE_MAX_LENGTH" ]
+  enc_type:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SECURITY_TYPE_MAX_LENGTH" ]
```

Comment:
These two should be enums instead of freeform strings.

I would even use `WSC::eWscAuth` directly - @tomereli @morantr  what do you think?

Reply:
Agree.

Reply:
Agree